<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicUserWindow</class>
 <widget class="QStackedWidget" name="MusicUserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicUserWindow</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="one">
   <widget class="QPushButton" name="userLogin">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>2</y>
      <width>45</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font-size:17px; border-none; color:#FFFFFF;background-color:transparent;}
QPushButton::hover{ background:rgba(255,255,255,50); border:none;}</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="two">
   <widget class="QLabel" name="userIcon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="userName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
